sycl/test-e2e/bindless_images/array/fetch_sampled_array.cpp

```diff
@@ -1,4 +1,6 @@
 // REQUIRES: aspect-ext_oneapi_image_array
+// UNSUPPORTED: target-amd
```

Comment:
same q on all of these on why they arent failing now

Reply:
The aspect `ext_oneapi_image_array` returns false when using the hip backend so this test does not run on AMD hardware. Same for L0 but that is planned to change soon.